CONTRIBUTING.md

```diff
@@ -177,6 +177,30 @@ To maintain understandable code and avoid circular dependencies it is important
 - **templates:** Golang templates for generating the html output.
 - **vendor:** External code that Gitea depends on.
 
+## API v1
+
+The API is documented by [swagger](http://gitea.com/api/swagger) and is based on [GitHub API v3](https://developer.github.com/v3/).
+Thus, Gitea´s API should use the same endpoints and fields as GitHub´s API as far as possible, unless there are good reasons to deviate. If GitHub doesn't provide...
```

Comment:
```suggestion
Thus, Gitea´s API should use the same endpoints and fields as GitHub´s API as much as possible, unless there are good reasons to deviate. If GitHub doesn't provide...
```

Also, what is `If GitHub doesn't provide...`? It seems like a half-finished thought, is there more that should go here?

Reply:
@jolheiser added + own suggestion ...